config/local/oracle.toml

```diff
@@ -19,7 +19,7 @@ production = false
     name = ""
     read_buffer_size = 0
     write_buffer_size = 0
-    handshake_timeout = "0s"
```

Comment:
Not that this will be over-written any time the oracle is started as it refreshes the configs.